<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VSCMainWindow</class>
 <widget class="QMainWindow" name="VSCMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>580</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>580</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Virtual Serial Port Controller</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/assets/png/vspclient_7.png</normaloff>:/assets/png/vspclient_7.png</iconset>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,90">
    <item>
     <widget class="QWidget" name="pnlButtons" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>12</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="btn01SPCreate">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="tabletTracking">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Create Serial Port</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn02SPRemove">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="tabletTracking">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Remove Serial Port</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn03LKCreate">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="tabletTracking">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Create Port Link</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn04LKRemove">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="tabletTracking">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Remove Port Link</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn05PortList">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="tabletTracking">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Get Port List</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn06LinkList">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="tabletTracking">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Get Link List</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn07Checks">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="tabletTracking">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Enable Checks</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn08Traces">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="tabletTracking">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Enable Traces</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn11SerialIO">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="tabletTracking">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Serial I/O Test</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn09Connect">
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="tabletTracking">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Connect</string>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn10Close">
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="tabletTracking">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>24</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>24</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="toolButton">
            <property name="maximumSize">
             <size>
              <width>12</width>
              <height>12</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="popupMode">
             <enum>QToolButton::DelayedPopup</enum>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonIconOnly</enum>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>14</width>
              <height>111</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="pnlContent" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>24</number>
       </property>
       <property name="leftMargin">
        <number>12</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="pnlPages" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QStackedWidget" name="stackedWidget">
            <property name="mouseTracking">
             <bool>true</bool>
            </property>
            <property name="tabletTracking">
             <bool>true</bool>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="currentIndex">
             <number>1</number>
            </property>
            <widget class="PGSPCreate" name="pg01SPCreate"/>
            <widget class="PGSPRemove" name="pg02SPRemove"/>
            <widget class="PGLKCreate" name="pg03LKCreate"/>
            <widget class="PGLKRemove" name="pg04LKRemove"/>
            <widget class="PGPortList" name="pg05PortList"/>
            <widget class="PGLinkList" name="pg06LinkList"/>
            <widget class="PGChecks" name="pg07Checks"/>
            <widget class="PGTrace" name="pg08Traces"/>
            <widget class="PGConnect" name="pg09Connect"/>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Results</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <item>
           <widget class="QTextBrowser" name="textBrowser">
            <property name="font">
             <font>
              <family>Menlo</family>
              <bold>true</bold>
             </font>
            </property>
            <property name="mouseTracking">
             <bool>false</bool>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="autoFormatting">
             <set>QTextEdit::AutoNone</set>
            </property>
            <property name="tabChangesFocus">
             <bool>true</bool>
            </property>
            <property name="lineWrapMode">
             <enum>QTextEdit::NoWrap</enum>
            </property>
            <property name="openExternalLinks">
             <bool>true</bool>
            </property>
            <property name="openLinks">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PGSPCreate</class>
   <extends>QWidget</extends>
   <header>pgspcreate.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PGSPRemove</class>
   <extends>QWidget</extends>
   <header>pgspremove.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PGLKCreate</class>
   <extends>QWidget</extends>
   <header>pglkcreate.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PGLKRemove</class>
   <extends>QWidget</extends>
   <header>pglkremove.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PGChecks</class>
   <extends>QWidget</extends>
   <header>pgchecks.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PGTrace</class>
   <extends>QWidget</extends>
   <header>pgtrace.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PGPortList</class>
   <extends>QWidget</extends>
   <header>pgportlist.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PGLinkList</class>
   <extends>QWidget</extends>
   <header>pglinklist.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PGConnect</class>
   <extends>QWidget</extends>
   <header>pgconnect.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>btn01SPCreate</tabstop>
  <tabstop>btn02SPRemove</tabstop>
  <tabstop>btn03LKCreate</tabstop>
  <tabstop>btn04LKRemove</tabstop>
  <tabstop>btn05PortList</tabstop>
  <tabstop>btn06LinkList</tabstop>
  <tabstop>btn07Checks</tabstop>
  <tabstop>btn08Traces</tabstop>
  <tabstop>btn11SerialIO</tabstop>
  <tabstop>btn09Connect</tabstop>
  <tabstop>btn10Close</tabstop>
  <tabstop>toolButton</tabstop>
  <tabstop>textBrowser</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btn10Close</sender>
   <signal>clicked()</signal>
   <receiver>VSCMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>365</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
